src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -96,6 +102,19 @@ public JobCollection get(Integer offset, Integer limit, String query) {
   @Transactional
   @Override
   public org.folio.des.domain.dto.Job upsertAndSendToKafka(org.folio.des.domain.dto.Job jobDto, boolean withJobCommandSend) {
```

Comment:
And with these changes is schedule logic will contine to trigger exports?
For example, we setup scheduling each day. After this we delete integration method. We should not expect to trigger this jog each day after it, did this logic implemented?

Reply:
@SerhiiNosko , 
added the logic for removing next schedule. please review.